docs/server/source/production-nodes/setup-run-node.md

```diff
@@ -0,0 +1,128 @@
+```text
+pip -V
+```
+
+If it says that `pip` isn't installed, or it says `pip` is associated with a Python version less than 3.4, then you must install a `pip` version associated with Python 3.4+. In the following instructions, we call it `pip3` but you may be able to use `pip` if that refers to the same thing. See [the `pip` installation instructions](https://pip.pypa.io/en/stable/installing/).
+
+On Ubuntu 16.04, we found that this works:
+```text
+sudo apt-get install python3-pip
+```
+
+That should install a Python 3 version of `pip` named `pip3`. If that didn't work, then another way to get `pip3` is to do `sudo apt-get install python3-setuptools` followed by `sudo easy_install3 pip`.
+
+You can upgrade `pip` (`pip3`) and `setuptools` to the latest versions using:
+```text
+pip3 install --upgrade pip setuptools
+pip3 -V
+```
+
+Now you can install BigchainDB Server (and officially-supported BigchainDB drivers) using:
+```text
+pip3 install bigchaindb
+```
+
+(If you're not in a virtualenv and you want to install bigchaindb system-wide, then put `sudo` in front.)
+
+Note: You can use `pip3` to upgrade the `bigchaindb` package to the latest version using `pip3 install --upgrade bigchaindb`.
+
+
+### How to Install BigchainDB Server from Source
+
+If you want to install BitchainDB from source because you want to use the very latest bleeding-edge code, clone the public repository:
+```text
+git clone git@github.com:bigchaindb/bigchaindb.git
+python setup.py install
+```
+
+
+## Configure BigchainDB Server
+
+Start by creating a default BigchainDB config file for a MongoDB backend:
+```text
+bigchaindb -y configure mongodb
+```
+
+(There's documentation for the `bigchaindb` command is in the section on [the BigchainDB Command Line Interface (CLI)](../server-reference/bigchaindb-cli.html).)
+
+Edit the created config file by opening `$HOME/.bigchaindb` (the created config file) in your text editor:
+
+* Change `"server": {"bind": "localhost:9984", ... }` to `"server": {"bind": "0.0.0.0:9984", ... }`. This makes it so traffic can come from any IP address to port 9984 (the HTTP Client-Server API port).
+* Change `"keyring": []` to `"keyring": ["public_key_of_other_node_A", "public_key_of_other_node_B", "..."]` i.e. a list of the public keys of all the other nodes in the cluster. The keyring should _not_ include your node's public key.
+* Ensure that `database.host` and `database.port` are set to the hostname and port of your MongoDB instance. (The port is usually 27017, unless you changed it.)
+
+For more information about the BigchainDB config file, see the page about the [BigchainDB configuration settings](../server-reference/configuration.html).
+
+
+## Maybe Update the MongoDB Replica Set
+
+**If this isn't the first node in the BigchainDB cluster**, then you must add your MongoDB instance to the MongoDB replica set. You can do so using:
```

Comment:
Also a note about _not_ starting BigchainDB before adding the new MongoDB instance to the replica set will be helpful.